.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: erlang
+otp_release:
+  - 21.0
+  - 20.3
+  - 19.3
+  - 18.3
+  - 17.5
+  - R16B03
```

Comment:
I like that we support this far aback, but maybe we should put a note that we aren't gonna fix buts on R16.